nemo/collections/asr/parts/submodules/multi_head_attention.py

```diff
@@ -157,9 +158,8 @@ def rel_shift(self, x):
             x (torch.Tensor): (batch, nheads, time, 2*time-1)
         """
         b, h, qlen, pos_len = x.size()  # (b, h, t1, t2)
-
-        # need to add a column of zeros on the left side of last two dimensions to perform the relative shifting
-        x = torch.nn.functional.pad(x, pad=(1, 0, 0, 0))  # (b, h, t1, t2+1)
+        # need to add a column of zeros on the left side of last dimension to perform the relative shifting
+        x = torch.nn.functional.pad(x, pad=(1, 0))  # (b, h, t1, t2+1)
```

Comment:
@VahidooX  please check this formula

Reply:
We discussed that yesterday on slack

Reply:
(1,0,0,0) is equivalent to (1,0) - just not misleading. Intent is to pad a single dimension